Core/GDCore/Extensions/Metadata/ValueTypeMetadata.h

```diff
@@ -192,6 +192,11 @@ class GD_CORE_API ValueTypeMetadata {
   static const gd::String numberType;
   static const gd::String stringType;
 
+  /**
+   * \brief Return the ValueType name from the property type.
```

Comment:
Could you add context about where those different types are used? Or a link to where the ValueType is defined/used?